src/components/ItaliaTheme/Footer/FooterSmall.jsx

```diff
@@ -83,16 +87,18 @@ const FooterSmall = () => {
               </li>
             ))}
           <li className="list-inline-item">
-            <button
+            <UniversalLink
               className="footer-gdpr-privacy-show-banner"
+              href="#"
               onClick={(e) => {
                 e.preventDefault();
                 dispatch(displayBanner(true, true));
               }}
               title={intl.formatMessage(messages.cookieSettings)}
+              data-element="privacy-policy-link"
             >
               {intl.formatMessage(messages.cookieSettings)}
-            </button>
+            </UniversalLink>
```

Comment:
Questa modifica non serve. Il link con `data-element="privacy-policy-link"` dovrà essere impostato tramite il pannello di controllo del subfooter, perché deve essere necessariamente il link diretto alla pagina della privacy policy. Le specifiche infatti dicono:

> Il link all’informativa sul trattamento dei dati personali **è presente nel footer del sito** e invia a una informativa sul trattamento dei dati personali

https://docs.italia.it/italia/designers-italia/app-valutazione-modelli-docs/it/versione-attuale/requisiti-e-modalita-verifica-comuni.html#criterio-c-si-3-3